distributions/openhab/src/main/resources/bin/update

```diff
@@ -353,6 +354,15 @@ rm -rf "${TempDir:?}"
 echo "Restoring previous file ownership ($FileOwner:$FileGroup)"
 chown -R "$FileOwner:$FileGroup" "$WorkingDir"
 
+## Start the upgrade tool
+echo "Starting JSON database update..."
+export OPENHAB_USERDATA="$OPENHAB_USERDATA"
+java -jar "$WorkingDir/runtime/bin/upgradetool.jar" || {
```

Comment:
Don't we need to add the commands to the command line? Or do you suggest we change the tool to always execute all steps? This is probably doable since we keep track of which steps have already run in the past.

Reply:
Yes, this was my assumption - that we can call the tool as many times as we want as it itself is aware of what it already did and what is still needed to do. I think this is the only way for it to react correctly, since we wouldn't want to update the update script here whenever there are new steps in the tool available.